### DISCUSSION TITLE
Question about installation

### DISCUSSION THREAD

**Discussion by foysalx** [iqm-finland/KQCircuits#98] [Q&A] [answered]
Hello, I am quite new to Klayout. Recently I installed KQcircuits as a saltpackage in Klayout. After I installed it, it didn't appear anywhere in Klayout or the libraries. I am using the recent version of Klayout on Windows 11 Pro. It would be really nice if I could get some help with it. 

Thanks a ton in advance!

Edit: Fixed it :D

**Comment by qpavsmi** [comment]
Hi, sorry I couldn't get to this earlier, I am currently on vacation so the response time for me may be slow :) Before we close this discussion thread, could you describe what was the issue that prevented you to use KQCircuits in KLayout and how you managed to fix it? This could help other future users if they experiece the same problem.

Also we designed the SALT installation to be used for demonstration and introduction purposes of KQCircuits. If you plan to do actual design work using KQCircuits, we highly recommend following the develop setup guide [here](https://iqm-finland.github.io/KQCircuits/developer/setup.html) and [here](https://iqm-finland.github.io/KQCircuits/developer/standalone.html)

**Reply by foysalx** [reply, marked as answer]
Hello! It sounds extremely stupid, but after looking at every possible option, I restarted it, and it appeared. I guess that for some reason if you don't restart certain elements of the package don't get installed, because after the restart it asked me to install missing elements of the package. 

Thanks if I already installed it using salt package, can I just follow the guidelines of the develop setup guide or do I need to remove it first?

**Reply by qpavsmi** [reply]
Thanks for reporting! I'll keep the restart in mind, maybe there is something we can do to make KQC work without restarting KLayout.

If you decide to install KQC using developer setup guide, it's better to remove the SALT installation since those two KQC installations may conflict.
